bundle-workflow/tests/tests_system/test_execute.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import subprocess
+import unittest
+from unittest.mock import patch
+
+from system.execute import execute
+
+
+class TestExecute(unittest.TestCase):
+    def test_execute_stdout(self):
+        (status, stdout, stderr) = execute("echo success", "/")  # (0, 'success\n', '')
+        self.assertEqual(status, 0)
+        self.assertEqual(stdout.strip(), "success")
+        self.assertEqual(stderr.strip(), "")
+
+    def test_execute_stderr(self):
+        (status, stdout, stderr) = execute(">&2 echo error", "/")  # (0, '', 'error\n')
```

Comment:
Add a check for the exit code, this could be a bug where you always get 0. Execute `exit 2` for example and check for 2 being returned.

Reply:
That is a good improvement let me check

Reply:
fixed.